README.md

```diff
@@ -1 +1,34 @@
-# hellpo-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-Microverse
+
+> A Code Review project.
+
+
+## Authors
+
+👤 **Alexander**
+
+- GitHub: [@Alexander](https://github.com/alexander16108)
+- Twitter: [@Alexander Mayowa](https://twitter.com/twitterhandle)
+- LinkedIn: [@Alexander](https://linkedin.com/in/codingrex)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
This MIT link is broken. Would you kindly fix it? Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.